docs/book/deployment.md

```diff
@@ -389,6 +390,37 @@ that will be used to call Lambda's [updateFunctionConfiguration](
 You can use this to modify the Lambda's timeout, change the memory size, or
 add your own environment variables. All sub keys are optional.
 
+#### DynamoDB TTL (optional)
+
+Litexa has limited support for setting a [Time To Live (TTL)](
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html)
+field with each database entry. This may be useful for limiting your table
+data to active users by automatically deleting old user data. To set a Time To
+Live specification, add the `dynamoDbConfiguration` object to your Litexa config:
+
+```javascript
+{
+  deployments: {
+    development: {
+      module: "@litexa/deploy-aws",
+      askProfile: "suncoast",
+      awsProfile: "prototyping",
+      dynamoDbConfiguration: {
+        timeToLive: {
+          AttributeName: "timeToLive", // needs to match the attribute name specified when enabling TTL
```

Comment:
Would use "ttl" since that's the attribute name example given in the AWS documentation, and to avoid possible confusion with the fact that the container property is also "timeToLive".